### PR TITLE
#5 :memo: Tranche 3 of documentation migration

### PR DESCRIPTION
**Pull request by d-ryan-ashcraft**
* [Data Delivery Pipelines](https://boozallen.github.io/aissemble/aissemble/current/data-delivery-pipeline-overview.html)
* [Machine Learning Pipeline](https://boozallen.github.io/aissemble/aissemble/current/machine-learning-pipeline-details.html)
* [Containers](https://boozallen.github.io/aissemble/aissemble/current/containers.html)
* [Testing](https://boozallen.github.io/aissemble/aissemble/current/testing.html)
* [Lineage Overview](https://boozallen.github.io/aissemble/aissemble/current/lineage-medatada-capture-overview.html)
* [Model Lineage](https://boozallen.github.io/aissemble/aissemble/current/model-lineage.html)
* [Data Lineage](https://boozallen.github.io/aissemble/aissemble/current/data-lineage.html)

### CODE REVIEW

**Review by castlebah — APPROVED** [review]
Clicked through the pages, verified links and skimmed content.  Good to merge

**Review by nartieri — COMMENTED** [review]
Substantive and administrative comments only. This looked really clean. Nice job!

File: docs/modules/ROOT/pages/data-delivery-pipeline-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+https://github.com/boozallen/aissemble/tree/dev/extensions/extensions-helm/aissemble-spark-application-chart[aissemble-spark-application ,role=external,window=_blank]
+Helm chart.
+[%collapsible]
+====
+****
+*Parameters*
+
+|===
+|*Name* | *Description*
+|applicationName
```
[the remark this answers is not in the document]
Comment (d-ryan-ashcraft) [reply docs/modules/ROOT/pages/data-delivery-pipeline-overview.adoc:76]: Agreed - removed the grey box in the collapsable section ✅

File: docs/modules/ROOT/pages/data-delivery-pipeline-overview.adoc [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+=== GET/invoke-pipeline/healthcheck
+A health check endpoint that returns http `200` if the invocation service is up and running.
+
+[%collapsible]
+====
+****
+*Parameters*
+
+|===
+|*Name* | *Description*
```
[the remark this answers is not in the document]
Comment (d-ryan-ashcraft) [reply docs/modules/ROOT/pages/data-delivery-pipeline-overview.adoc:118]: Agreed - removed the grey box in the collapsable section ✅

File: docs/modules/ROOT/pages/machine-learning-pipeline-details.adoc [hunk 3]
```diff
@@ -0,0 +1,569 @@
...
+		}
+	]
+}
+----
+
+//todo section seems redundant with previous documentation on the mda generation build-action-build-action loop
+=== POM
+After specifying the contents of the JSON file, you would then run a `mvn clean install` to generate the associated POM
+file. An example POM file associated with the example JSON from above is shown below. (some content in the pom file
+will be specific to the machine learning pipeline defined for the project)
```
[the remark this answers is not in the document]
Comment (d-ryan-ashcraft) [reply docs/modules/ROOT/pages/machine-learning-pipeline-details.adoc:53]: ✅ UPDATED

File: docs/modules/ROOT/pages/machine-learning-pipeline-details.adoc [hunk 4]
```diff
@@ -0,0 +1,569 @@
...
+|`pyproject.toml`
+|All Python projects emitted by aiSSEMBLE are https://python-poetry.org/[Poetry,role=external,window=_blank] projects
+that rely on https://github.com/TechnologyBrewery/habushu[Habushu,role=external,window=_blank] for DevOps automation.
+`pyproject.toml` is a required Poetry configuration that aligns with
+https://peps.python.org/pep-0518/[PEP-518,role=external,window=_blank] and defines the project's build system
+requirements.  Developers should customize this configuration as needed to support additional dependencies or
+capabilities required within pipelines.
+
+|`src/aissemble_machine_learning_inference/validation/inference_payload_definition.py`
+|Defines the request/response payloads that captures that format of input data records for inferencing as well as the
```
[the remark this answers is not in the document]
Comment (d-ryan-ashcraft) [reply docs/modules/ROOT/pages/machine-learning-pipeline-details.adoc:195]: ✅ Definitely an improvement

File: docs/modules/ROOT/pages/machine-learning-pipeline-details.adoc [hunk 5]
```diff
@@ -0,0 +1,569 @@
...
+
+In order to include this API in your project, include an execution in your deployment `pom.xml` pointing to the
+`training-deploy` profile. More information is available on the xref:containers.adoc#_containers[Container Support page].
+
+Here are the available routes:
+
+POST /training-jobs?pipeline_step={pipelineStep}
+
+* `pipelineStep` is the name of the "training" ML pipeline step you would like to execute
+** Must be CamelCased
```
[the remark this answers is not in the document]
Comment (d-ryan-ashcraft) [reply docs/modules/ROOT/pages/machine-learning-pipeline-details.adoc:399]: Good question - I'm not sure, but we do tend to want this stuff in CamelCase.  That said, it should probably cause and error when reading the metamodel to help ensure this is smooth.

File: docs/modules/ROOT/pages/machine-learning-pipeline-details.adoc [hunk 6]
```diff
@@ -0,0 +1,569 @@
...
+            " 'succeeded': None,\n"
+            " 'uncounted_terminated_pods': {'failed': None, 'succeeded': "
+            'None}}'}]
+----
+
+
+GET /training-jobs?pipeline_step={pipelineStep}
+
+* `pipelineStep` is the name of the "training" ML pipeline step you would like to execute
+** Must be CamelCased
```
[the remark this answers is not in the document]
Comment (d-ryan-ashcraft) [reply docs/modules/ROOT/pages/machine-learning-pipeline-details.adoc:446]: I think this is correct, but we can definitely test it.

**Comment by d-ryan-ashcraft** [comment]
Thanks @nartieri for the detailed read!  All changes incorporated.

**Pull request merged**